crates/wadm/src/lib.rs

```diff
@@ -39,3 +73,401 @@ pub const APP_SPEC_ANNOTATION: &str = "wasmcloud.dev/appspec";
 pub const SCALER_KEY: &str = "wasmcloud.dev/scaler";
 /// The default link name. In the future, this will likely be pulled in from another crate
 pub const DEFAULT_LINK_NAME: &str = "default";
+pub(crate) const WADM_EVENT_STREAM_NAME: &str = "wadm_events";
```

Comment:
Super Nit: Should we just make these full `pub` and preface them with `DEFAULT_`? It might be useful to have them around if consuming as a library